src/elements/dialogs/session-details.html

```diff
@@ -156,6 +156,10 @@ <h3>{$ sessionDetails.speakers $}</h3>
 
       _close() {
         dialogsActions.closeDialog(DIALOGS.SESSION);
+        this.dispatchEvent(new CustomEvent('reset-query-params', {
```

Comment:
Where is this event listened to?

Reply:
Here is a listener: https://github.com/gdg-x/hoverboard/blob/hoverboard-v2/src/hoverboard-app.html#L341